File: 69-sqrtx/sqrtx.go
func mySqrt(x int) int {
    if x==0 || x ==1 {
        return x
    }
    r:=float64(x)
    rx:=0.5*(r+float64(x)/r)
    for r!=rx {
        r = rx
        rx = 0.5*(r+float64(x)/r)
    }
    return int(r)
}

File: 69-sqrtx/sqrtx.py
class Solution:
    def mySqrt(self, x: int) -> int:
        if x == 0 or x ==1 :
            return x
        r =x
        rx = 0.5*(r+x/r)
        while(r!=rx):
            r= rx
            rx = 0.5*(r+x/r)
        return int(r)